### ISSUE TITLE
FHIMP - Centralize location of flood hazard maps

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#306]
Suggestion as to how the sector can make flood mapping / flood risk sites more relevant for users: Many/most flood hazard maps are located across multiple locations - there is an opportunity to centralize some of them or use consistent standards.

**Comment by JeffreyAshtonNRCan** [comment]
Flood hazard maps from multiple locations could be added to the Flood Map and Dataset Gallery page, or a standalone gallery page could be created.  Need to identify flood hazard maps and their location.

Resources identified in Phase I, Milestone 2 report:

Historical Flood Events Open Government - Historical Flood Events (HFE) Portal
National Hydrographic Network Open Government - National Hydro Network - NHN - GeoBase Series
Canada Flood Map Inventory Open Government - Canada Flood Map Inventory (CFM)
Floods in Canada Open Government - Floods in Canada - Cartographic Product Collection
Floods and River Ice Break-up NRCan Floods and River Ice Break-up
Flood Susceptibility Index Open Government - Flood Susceptibility Index (FSI)
High Resolution Digital Elevation Model (HRDEM) Open Government - High Resolution Digital Elevation Model (HRDEM) - CanElevation Series
2020 Land Cover of Canada Open Government - 2020 Land Cover of Canada
Automatically Extracted Buildings Open Government - Automatically Extracted Buildings
The Open Database of Buildings Stats Can - The Open Database of Buildings
Climate Data ECCC and partners - climatedata.ca

**Comment by JeffreyAshtonNRCan** [comment]
FHIMP to contact provinces

**Comment by sean-eagles** [comment]
Yes, it would be interesting to harmonise and centralize flood hazard maps. However, these maps are a provincial/territorial jurisdiction so we don’t control how the PTs decide to produce their maps.
